driver-core/src/main/com/mongodb/internal/connection/ZstdCompressor.java

```diff
@@ -27,6 +27,14 @@
 import java.util.List;
 
 class ZstdCompressor extends Compressor {
+    ZstdCompressor() {
+        try {
+            Class.forName("com.github.luben.zstd.Zstd");
```

Comment:
Alternatively, I could do `com.github.luben.zstd.util.Native.load()` but it feels like an implementation detail of `zstd-jni`. WDYT?